src/promptflow-devkit/promptflow/_proxy/_python_inspector_proxy.py

```diff
@@ -5,6 +5,7 @@
 from promptflow._constants import FlowEntryRegex
 from promptflow._core.entry_meta_generator import _generate_flow_meta
 from promptflow._sdk._constants import FLOW_META_JSON_GEN_TIMEOUT
+from promptflow._sdk._utilities.general_utils import is_eval_batch_run
```

Comment:
may add some PR description on stats number of the improvement

Reply:
Added.